variation/validators/amino_acid_insertion.py

```diff
@@ -77,6 +81,10 @@ def get_valid_invalid_results(self, classification_tokens, transcripts,
         :param dict mane_data_found: MANE Transcript information found
         :param bool is_identifier: `True` if identifier is given for exact
             location. `False` otherwise.
+        :param str hgvs_dup_del_mode: Must be: `default`, `cnv`,
+            `repeated_seq_expr`, `literal_seq_expr`.
+            This parameter determines how to represent HGVS dup/del expressions
+            as VRS objects.
```

Comment:
this shows up in a few places -- is there room to make it an enum like the others?

Reply:
So there is an enum [here](https://github.com/cancervariants/variation-normalization/blob/hgvs-dup-del-mode/variation/schemas/normalize_response_schema.py#L10). Initially, I was setting the type to `HGVSDupDelMode` but I was getting warnings about it being a string. Might be something on my end, so I can switch.

Reply:
Oh wait. Scratch that. I forgot I did end up switching but this one must've slipped through. Thanks!